io.c

```diff
@@ -1209,8 +1214,7 @@ io_flush_buffer_async2(VALUE arg)
     rb_io_t *fptr = (rb_io_t *)arg;
     VALUE ret;
 
-    ret = (VALUE)rb_thread_call_without_gvl2(io_flush_buffer_sync2, fptr,
-					     RUBY_UBF_IO, NULL);
+    ret = (VALUE)rb_thread_call_without_gvl2(io_flush_buffer_sync2, fptr, RUBY_UBF_IO, NULL);
```

Comment:
Is this just deleting a newline, or you have some other intention?

Reply:
Most other code in that file does not follow any convention for wrapping and it was causing me trouble so I deleted newline.

Reply:
OK, was just confirmation.  Not always a good idea to mix indent changes into something else, though.